nautobot/docs/plugins/development.md

```diff
@@ -148,34 +126,34 @@ Nautobot looks for the `config` variable within a plugin's `__init__.py` to load
+| Name | Description |
+| ---- | ----------- |
+| `author` | Name of plugin's author |
+| `author_email` | Author's public email address |
+| `base_url` | (Optional) Base path to use for plugin URLs. If not specified, the project's `name` will be used. |
+| `caching_config` | Plugin-specific cache configuration |
+| `custom_validators` | The dotted path to the list of custom validator classes (default: `custom_validators.custom_validators`) |
+| `datasource_contents` | The dotted path to the list of datasource (Git, etc.) content types to register (default: `datasources.datasource_contents`) |
+| `default_settings` | A dictionary of configuration parameters and their default values |
+| `description` | Brief description of the plugin's purpose |
+| `graphql_types` | The dotted path to the list of GraphQL type classes (default: `graphql.graphql_types)` |
+| `installed_apps` | A list of additional Django application dependencies to automatically enable when the plugin is activated (you must still make sure these underlying dependent libraries are installed) |
+| `jobs` | The dotted path to the list of Job classes (default: `jobs.jobs`) |
+| `max_version` | Maximum version of Nautobot with which the plugin is compatible |
+| `menu_items` | The dotted path to the list of menu items provided by the plugin (default: `navigation.menu_items`) |
+| `middleware` | A list of middleware classes to append after Nautobot's built-in middleware |
+| `min_version` | Minimum version of Nautobot with which the plugin is compatible |
+| `name` | Raw plugin name; same as the plugin's source directory |
+| `required_settings` | A list of any configuration parameters that **must** be defined by the user |
+| `template_extensions` | The dotted path to the list of template extension classes (default: `template_content.template_extensions`) |
+| `verbose_name` | Human-friendly name for the plugin |
+| `version` | Current release ([semantic versioning](https://semver.org/) is encouraged) |
 
 All required settings must be configured by the user. If a configuration parameter is listed in both `required_settings` and `default_settings`, the default setting will be ignored.
 
 ### Install the Plugin for Development
 
-To ease development, it is recommended with the virtual environment activated to use `poetry install` command from the location of the `pyproject.toml` file. By default Poetry installs the current project as editable and the dependencies as well.
+To ease development, it is recommended with the Nautobot virtual environment activated (`/opt/nautobot/bin/activate` if using official doc). Then while in the location of the `pyproject.toml` file execute the command `poetry install` to install the local package. By default Poetry installs the current project as editable and the dependencies as well.
```

Comment:
The first two sentences are confusing to me because we're saying to activate the virtualenv that appears to be in the production-deployed path of `/opt/nautobot`, but then also directing the reader to run `poetry install`?  Can you please clarify this?

The final sentence on a 2nd read through is a bit hard to grok. Also, some might not know what "editable" mode means. Can we reword this, please? Here's a suggestion:

```suggestion
To ease development, it is recommended with the Nautobot virtual environment activated (`/opt/nautobot/bin/activate` if using official doc). Then while in the location of the `pyproject.toml` file execute the command `poetry install` to install the local package. Poetry installs the current project and its dependencies in editable mode (aka ["development mode"](https://setuptools.readthedocs.io/en/latest/userguide/development_mode.html)).
```